test/e2e/spec/api_spec.js

```diff
@@ -379,6 +379,38 @@ describe("API v1", function () {
                             }
                         )).to.have.status(400);
                     });
+
+                    describe("even in subsequent requests", function () {
+                        before(function() {
+                            return expect(chakram.post(
+                                '/domains/' + domain + '/rrsets/',
+                                {
+                                    'subname': 'duplicate-contents', 'type': 'AAAA',
+                                    'records': ['::1'], 'ttl': 60
+                                }
+                            )).to.have.status(201);
+                        });
+
+                        it("still does not accept a semantic duplicate", function () {
+                            return expect(chakram.post(
+                                '/domains/' + domain + '/rrsets/',
+                                {
+                                    'subname': 'duplicate-contents', 'type': 'AAAA',
+                                    'records': ['::0001'], 'ttl': 60
+                                }
+                            )).to.have.status(400);
+                        });
+
+                        it("still does not accept a semantic duplicates", function () {
+                            return expect(chakram.post(
+                                '/domains/' + domain + '/rrsets/',
+                                {
+                                    'subname': 'duplicate-contents', 'type': 'AAAA',
+                                    'records': ['::1', '::0001'], 'ttl': 60
```

Comment:
This should be rejected regardless of `::1` being created in `before()`, because duplicate content in an RRset is not allowed even when fresh. Are we testing that as well?

Reply:
yes, in lines 368 to 381